### ISSUE TITLE
Why is my training loss value 140000000?

### ISSUE THREAD

**Issue by bbbbb12345** [leegaeun/CT_Kernel_Conversion#1]
Is this loss value Normal? I modified the original file(convertor_to_lmdb), read the dicom CT image. In DIGITS ,i set subtract mean as "image". Thanks!

**Comment by leegaeun** [comment]
Yes, the loss function of this model is "EuclideanLoss", which is the sum of squares of difference. For example, if you train a 512x512 image with batchsize 1, the loss value 140000000 can be seen as a loss of approximately 23 per pixel. (140000000/(512*512))^1/2   ~= 23

**Comment by bbbbb12345** [comment]
Got it, thanks for your explanation！
I'm still a little confused about a code line in "converter_to_lmdb.py" : 
                                  "mean_image = (image_sum / image_count).astype('uint16')". 
Since range of pixel values is about -1024 to 2XXX（Hounsfield）in image that I readed in Dicom format, many pixels have negative values. So I replaced "uint16" with "int16", not sure if i am doing it right.

**Comment by leegaeun** [comment]
Oh, your correction is right. The code has been modified, thank you.
